packages/analytics-js/src/components/userSessionManager/UserSessionManager.ts

```diff
@@ -262,6 +267,26 @@ class UserSessionManager implements IUserSessionManager {
     }
   }
 
+  /**
+   * A function to make an external request to set the cookie from server side
+   * @param key       cookie name
+   * @param value     encrypted cookie value
+   */
+  setServerSideCookie(key: string, value: string): void {
+    let baseUrl = state.lifecycle.activeDataplaneUrl.value;
+    if (typeof state.loadOptions.value.cookieServerUrl === 'string') {
+      baseUrl = state.loadOptions.value.cookieServerUrl;
+    }
+    this.httpClient?.getAsyncData({
+      url: `${baseUrl}/setCookie`,
```

Comment:
Ensure that we're not adding `/` if it's already there.

Reply:
Use the URL utility functions to handle it smoothly.

Reply:
Addressed